.github/FUNDING.yml

```diff
@@ -1,6 +1,6 @@
 # These are supported funding model platforms
 
-github: mrtnzlml # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
+github: [mrtnzlml, tbergq] # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
```

Comment:
@tbergq important part of this comment is **Sponsors-enabled usernames** so this won't work. It results in the following error when you click on the "Sponsor" button:

<img width="414" alt="Screenshot 2020-10-25 at 08 27 59" src="https://user-images.githubusercontent.com/978368/97109975-bb880c80-16d6-11eb-9717-a89c7b95342a.png">


Could you please remove yourself until you go through the process? Feel free to add yourself again when you are approved.